python/veles/compatibility/six.py

```diff
@@ -0,0 +1,868 @@
+"""Utilities for writing code that runs on Python 2 and 3"""
```

Comment:
So we a) abandon generating C++ file on build, and put generated file directly into repository or b) require user to setup proper Python environment with required packages before building C++ code ?